configs/infer_config.yaml

```diff
@@ -8,6 +8,28 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
+### Data Pre-processing parameters
+## Generate intervals files
+interval_size: 50000
+wg: False
```

Comment:
I think we could get rid of the wg option, see  previous comment.

Reply:
done